lib/isInvalidTokenError.js

```diff
@@ -1,4 +1,4 @@
-export function checkError({e}) {
+export function isInvalidTokenError({e}) {
```

Comment:
Despite this being a pretty long name, I checked the other public APIs and it looks like we use `AccessToken` verbiage so we should be consistent here too:

```suggestion
export function isInvalidAccessTokenError({e}) {
```

Reply:
Updated.